data/templates/install-coreos.sh

```diff
@@ -2,4 +2,5 @@
 set -e
 curl -O http://<%=server%>:<%=port%>/api/current/templates/pxe-cloud-config.yml
 sudo coreos-install -d <%=installDisk%> -c pxe-cloud-config.yml -b <%=repo%>
+wget --spider http://<%=server%>:<%=port%>/api/current/templates/<%=completionUri%>
```

Comment:
Happy to change it, though.  curl is more ubiquitous, and your method will work just fine! Let me know what you think.